src/Illuminate/View/Concerns/ManagesLayouts.php

```diff
@@ -146,11 +146,7 @@ protected function extendSection($section, $content)
      */
     public function yieldContent($section, $default = '')
     {
-        $sectionContent = $default instanceof View ? $default : e($default);
-
-        if (isset($this->sections[$section])) {
-            $sectionContent = $this->sections[$section];
-        }
+        $sectionContent = $this->sections[$section] ?? ($default instanceof View ? $default : e($default));
```

Comment:
Revert this. This change makes the code harder to read than it is now.

Reply:
Reverted.